### PR TITLE
Feature/scat 2089 respond to supplier

### PR DESCRIPTION
**Pull request by venkibric**
### Change description ###

Added code for responding message and reformatted MessageServiceTest Class

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

### CODE REVIEW

File: src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/MessageService.java [hunk 1]
```diff
@@ -84,13 +89,21 @@ public String sendOrRespondMessage(final String profile, final Integer projectId
...
-        .messageClassification(nonOCDS.getClassification().getValue())
-        .senderName(buyerUser.get().getName()).supplierName("").messageReceivedDate("");
+        .messageClassification(nonOCDS.getClassification().getValue()).senderName("")
+        .supplierName("").messageReceivedDate("");
 
     // To reply the message
     if (nonOCDS.getParentId() != null) {
-      // TODO get message details using parentId
-      inputBuilder.messagingAction(RESPOND_MESSAGE).messageReceivedDate("message-receive-date");
+      DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern(DATE_FORMATTER);
```
Comment (tommyb82) [inline src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/MessageService.java:97]: Can we make the formatter instance a class constant - e.g.
`static final DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'.000+00:00'")`
Do any of the predefined formatters meet the required format spec?

**Review by tommyb82 — APPROVED** [review]
Test class has been entirely reformatted so difficult to see what's changed - just that one comment around the formatter instance.

File: src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/MessageService.java (hunk 1 above)
Comment (venkibric) [reply src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/MessageService.java]: Created formatter instance at class level.

**Comment by venkibric** [comment]
>

**Pull request merged**